Ecommerce/index.css

```diff
@@ -1 +1,78 @@
-/* Write code below for ECOMMERCE TASK */
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>About Us</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <style>

```

Comment:
The class selector '.top' should be renamed to '.top-container' or something similar to accurately represent its purpose.